### PR TITLE
NETOBSERV-601: Added workaround documentation

### PR DESCRIPTION
**Pull request by OlivierCazade**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -321,6 +321,23 @@ kubectl get pods -n netobserv -l app=netobserv-plugin
 kubectl logs -n netobserv -l app=netobserv-plugin
 ```
 
+
+It is also possible that the openshift console is failing to restart:
```
Comment (jotak) [inline README.md:325]: I would mention the link to the known issue:

```suggestion
It is also possible that the OpenShift Console is failing to restart. There is for instance a [known issue](https://issues.redhat.com/browse/OCPBUGS-810) with very small clusters, causing this problem.
```

**Review by jpinsonneau — APPROVED** [review]
LGTM, maybe just remind the command to quickly kill console pods ?

File: FAQ.md [hunk 2]
```diff
@@ -137,8 +137,24 @@ kubectl get pods -n netobserv -l app=netobserv-plugin
...
+```
+NAME                         READY   STATUS    RESTARTS   AGE
+console-65486f987c-bk6zn     1/1     Running   0          44m
+console-67fd7b6f49-l5nmq     0/1     Pending   0          10m
+console-67fd7b6f49-wmhp4     0/1     Pending   0          10m
+downloads-5fc6cc467f-2fttx   1/1     Running   0          45m
+downloads-5fc6cc467f-2gfpw   1/1     Running   0          45m
+```
+
+To force a restart kill the running console pod, note that this will make the console anavailable for some time.
```
[the remark this answers is not in the document]
Comment (jotak) [reply FAQ.md:154]: also typo anavailable => unavailable

**Pull request merged**